handlers/handler.go

```diff
@@ -0,0 +1,56 @@
+package handlers
+
+import (
+	"net/http"
+	"github.com/ONSdigital/dp-csv-splitter/aws"
+	"io/ioutil"
+	"encoding/json"
+	"fmt"
+	"io"
+)
+
+// URIParamMissingMsg error message for URI parameter missing from request parameters
+const URIParamMissingMsg = "Please specify a uri parameter."
+const URIParamName = "uri"
+const SuccessMsg = "Your file has been sent to the Chopper"
+const ReadRequestBodyErrMsg = "Could not read body"
+
+var AWSCli aws.AWSClient
+
+type ByteSliceReader func(r io.Reader) ([]byte, error)
+
+var requestBodyReader ByteSliceReader = ioutil.ReadAll
+
+
+// Handle TODO
```

Comment:
What is it that we need `TODO`, seems like we've got enough here to in terms of querying AWS and getting the file but we don't return the file. Is my assumption here correct?